week_1/Project/youtube-fe/youtube.html

```diff
@@ -0,0 +1,200 @@
+                    </p>
+                    <p class="video-author">
+                        Kevin Colaco
+                    </p>
+                    <p class="video-stats">
+                        3.4M views &#183; 6 months ago
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="video-preview">
+            <div class="thumbnail-row">
+                <img class="thumbnail" src="thumbnails/website_yt_thumbnail.webp" alt="React Thumbnail">
+                <div class="video-time">6:48</div>
+            </div>
+            <div class="video-info-grid">
+                <div class="channel-image">
+                    <img class="profile-picture" src="channel-images/ssd_channel.jpg">
+                </div>
+                <div class="video-info">
+                    <p class="video-title">
+                        How to put an HTML website online
+                    </p>
+                    <p class="video-author">
+                        SuperSimpleDev
+                    </p>
+                    <p class="video-stats">
+                        1.3M views &#183; 1 year ago
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="video-preview">
+            <div class="thumbnail-row">
+                <img class="thumbnail" src="thumbnails/algo_yt_thumbnail.webp" alt="React Thumbnail">
+                <div class="video-time">27:04</div>
+            </div>
+            <div class="video-info-grid">
+                <div class="channel-image">
+                    <img class="profile-picture" src="channel-images/wired_channel.jpg">
+                </div>
+                <div class="video-info">
+                    <p class="video-title">
+                        Explaining Algorithms in 5 levels
+                    </p>
+                    <p class="video-author">
+                        WIRED
+                    </p>
+                    <p class="video-stats">
+                        1.2M views &#183; 2 months ago
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="video-preview">
+            <div class="thumbnail-row">
+                <img class="thumbnail" src="thumbnails/ffc_yt_thumbnail.webp" alt="React Thumbnail">
+                <div class="video-time">1:24:37</div>
+            </div>
+            <div class="video-info-grid">
+                <div class="channel-image">
+                    <img class="profile-picture" src="channel-images/ffc_channel.jpg">
+                </div>
+                <div class="video-info">
+                    <p class="video-title">
+                        Backend Development
+                    </p>
+                    <p class="video-author">
+                        freeCodeCamp.org
+                    </p>
+                    <p class="video-stats">
+                        5M views &#183; 1 year ago
```

Comment:
@codal-kcolaco `&#183` what is this called as?

Reply:
@abhi-codal  `&#183` is html coded character set. This particular element defines a dot that is needed for Youtube clone.